src/consensus/vote.cpp

```diff
@@ -368,12 +382,10 @@ std::vector<Vote> VoteManager::getVerifiedVotes(uint64_t const pbft_round, size_
   for (auto const& v : future_unverifiable_votes) {
     removeUnverifiedVote(v.getRound(), v.getHash());
   }
-
-  return getVerifiedVotes();
 }
 
 // cleanup votes < pbft_round
-void VoteManager::cleanupVotes(uint64_t pbft_round) {
+void VoteManager::cleanupVotes(uint64_t const& pbft_round) {
```

Comment:
please remove `const&` it is not necessary and it could be slower then just passing a copy of uint64_t

Reply:
Yeah, you right. Fixed, thanks